tests/Endpoint/PartitionTest.php

```diff
@@ -338,6 +365,12 @@ public function signatureVersionProvider()
                         'puff' => [],
                     ],
                 ],
+                'sdb' => [
```

Comment:
Great, it would also help reduce the time when `changelogBuilder` is trying to replacing string : )